src/lib/components/SearchBar/SearchBarILS.js

```diff
@@ -65,13 +67,15 @@ SearchBarILS.propTypes = {
   onKeyPressHandler: PropTypes.func,
   onPasteHandler: PropTypes.func,
   onSearchHandler: PropTypes.func.isRequired,
+  onChangeHandler: PropTypes.func,
   placeholder: PropTypes.string,
   className: PropTypes.string,
 };
 
 SearchBarILS.defaultProps = {
   onKeyPressHandler: null,
   onPasteHandler: null,
```

Comment:
if you set the default onPasteHandler as `() => {}` instead of setting it in the components that use the SearchBarILS?

Reply:
`SearchBarILS` provides a default handler if none is defined, and it's the one that trigger search when <kbd>Enter</kbd> is pressed. Could be refactored to have an additional props that controls this (ie. `isSearchOnPaste`).